pipeline/dags/greenhouse_xco2.py

```diff
@@ -7,14 +7,12 @@
 # layer
 LAYER_ID = 'greenhouse'
 LAYER_VARIABLE = 'xco2'
-RESOLUTION = '180 90'
-EXTEND = '-180 -90 180 90'
 METADATA = {
     "id": f'{LAYER_ID}.{LAYER_VARIABLE}',
     "timestamps": [],  # will be injected
     "min_value": 0.0003599192132242024,
     "max_value": 0.0004179562965873629,
-    "type": "image",  # 'tiles' or 'image'
+    "type": "tiles",  # 'tiles' or 'image'
```

Comment:
Maybe we can even keep `"type": "image"` here, because of the low resolution of the dataset